### ISSUE TITLE
Feature Request: hide shorts from "uploads" on channels, or completely

### ISSUE THREAD

**Issue by jeyjai** [yuliskov/SmartTube#2408]
Hey, is it possible to hide shorts from the regular uploads when browsing a channel? They have their own section if you scroll down a bit on their channel page, past playlists and such, and if it's not possible to hide one and leave the other, maybe it's possible to at least have the option to hide them completely when browsing someone's channel?

**Comment by Ed254** [comment]
Found this today but I'm not sure if it's universal. Please try it and give feedback.
Go to settings, general, misc, tick hide shorts everywhere.

**Comment by LyveMyche** [comment]
YES! Thank you for adding this! its SOOO much nicer browsing my subscriptions without having to weed through all of the shorts...

**Comment by mirko77** [comment]
18.24, Shorts are hidden but they are still played when playing all the videos of a channel in sequence.
Is that intended behaviour?
